01_julia_intro/01_julia_intro.jl

```diff
@@ -0,0 +1,475 @@
+### A Pluto.jl notebook ###
+# v0.12.18
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 0a4fc5fc-544d-11eb-0189-6b1c959b1eb1
+md"
+# Meeting Julia
+Julia is a young, free, open-source and promising general-purpose language, designed and developed by Jeff Bezason, Alan Edelman, Viral B. Shah and Stefan Karpinski in MIT. 
+For years, programming languages were limited to either having a simple syntax, being readable, and having good abstraction capabilities, or being designed for technical, high-performance and resource-intensive computations. This led applied scientists to face the task of not only learning two different languages –one high-level, the other low-level-, but also learning how to have them communicate with one another, which is far from trivial in many cases. This is what Julia creators called the *two-language problem*, and it costs programmers and scientists valuable time and effort which may be better invested in solving the actual problems that they needed to compute. Julia is designed to bridge the gap, as it is created from scratch to be both fast and easy to understand, even for people who are not programmers or computer scientists.
+	Julia is dynamically typed and is great for interactive use. It also uses multiple dispatch as a core design concept, which adds to the composability of the language. In conventional, single-dispatched, object-oriented programming languages, when invoking a method, one of the arguments has a special treatment since it determines which of the methods contained in a function is going to be applied. Multiple dispatch is a generalization of this for all the arguments of the function, so the method applied is going to be the one that matches exactly the number of type of the function call.
+"
+
+# ╔═╡ 292d20ea-5a8e-11eb-2a96-a37689e468ca
+md"
+## First steps into the Julia world
+As with every programming language, it is useful to know some of the basic operations and functionalities. We encourage you to open a Julia session REPL and start experimenting with all the code written in this chapter to start developing an intuition about the things that make Julia code special.
+
+The common arithmetical and logical operations are all available in Julia:
+
+* $ + $: Add operator
+* $ - $: Substract operator
+* $ * $: Product operator
+* $ / $: Division operator
+
+Julia code is intended to be very similar to math. So insted of doing something like
```

Comment:
This instance of "insted" has been corrected, but I found another one here:

 `Later we will be introducing matrices and, naturally, a 2 will appear in this place insted a 1.`

it should say "instead of a 1"